src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -217,6 +217,10 @@ export interface SavedObjectsServiceStart {
    * {@link SavedObjectsType | saved object types}
    */
   getTypeRegistry: () => ISavedObjectTypeRegistry;
+  /**
+   * Returns the maximum number of objects allowed for import or export operations.
+   */
+  getImportExportObjectLimit: () => number;
```

Comment:
I exposed it in the `setup` contract because that's a value that could be needed from the plugin's `setup` phase. 
We usually don't want to expose APIs in both `setup` and `start` methods (even if we do have exceptions to that rule)

Now that's it's done however, I think it is alright, unless someone says otherwise. @rudolf ?

Reply:
> We usually don't want to expose APIs in both setup and start methods (even if we do have exceptions to that rule)

Ah, I knew we had several examples in `core` where we did this, but didn't realize it was an exceptional case, and not a pattern to be repeated. Thanks for the heads-up!

Reply:
In general, we want there to be one way and one way only to do things with Core API's to minimise confusion. This makes sense for API methods that are already bound to a specific lifecycle, but I think we can possibly be less strict with read-only config values.

However in this case I don't think it should impact the code quality a lot, so I'll leave another comment for a suggested way to refactor it.